cvat-ui/src/components/annotation-page/annotation-filter/annotation-filter.enum.ts

```diff
@@ -0,0 +1,100 @@
+// Copyright (C) 2020-2021 Intel Corporation
+//
+// SPDX-License-Identifier: MIT
+
+export enum StateLevels {
+    concatenator,
+    filterBy,
+    operator,
+    value,
+    attribute,
+    attributeOperator,
+    attributeValue,
+    anotherAttributeLabel,
+    anotherAttributeValue,
+}
+
+export enum ActionType {
+    concatenator,
+    filterBy,
+    operator,
+    value,
+    attribute,
+    attributeOperator,
+    attributeValue,
+    anotherAttributeLabel,
+    anotherAttributeValue,
+    fillState,
+    partialReset,
+    reset,
```

Comment:
I expect all actions to be a verb (or contain a verb)
For example I cannot understand what such actions like ``concatenator``, ``value``, ``operator``, ``attribute``, etc. do.
Also to follow the current codestyle let actions to be written in upper case: FILTER_BY, FILL_STATE, PARTIAL_RESET.